jenkins/map.jinja

```diff
@@ -5,6 +5,14 @@
         'group': 'jenkins',
         'home': '/var/lib/jenkins',
         'port': '80',
-        'server_name': None
+        'server_name': None,
```

Comment:
Agreed, the code will be more portable this way !

On Fri, Oct 9, 2015 at 10:30 PM, Devon Stewart <notifications@github.com>
wrote:

> In jenkins/map.jinja
> <https://github.com/saltstack-formulas/jenkins-formula/pull/31#discussion_r41674704>
> :
>
> > @@ -5,6 +5,14 @@
> >          'group': 'jenkins',
> >          'home': '/var/lib/jenkins',
> >          'port': '80',
> > -        'server_name': None
> > +        'server_name': None,
>
> Yeah, sorry, haven't had a chance to look at it yet; if it's all the same
> to you, I'd rather fetch the blessed version from Jenkins' service and
> cache it somewhere we choose
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/saltstack-formulas/jenkins-formula/pull/31/files#r41674704>
> .
>



-- 
http://yourlabs.org <http://blog.yourlabs.org>
Customer is king - Le client est roi - El cliente es rey.